<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ForgotPass</class>
 <widget class="QDialog" name="ForgotPass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>160</y>
     <width>241</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_user_name">
          <property name="text">
           <string>user name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_fav_animal">
          <property name="text">
           <string>fav anima</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_fav_city">
          <property name="text">
           <string>fav city</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_fav_color">
          <property name="text">
           <string>fav color</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLineEdit" name="lineEdit_get_user_name"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_get_fav_animal"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_get_fav_city"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_get_fav_color"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>check</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>200</y>
     <width>203</width>
     <height>134</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QLabel" name="label_pass_status">
      <property name="text">
       <string>status</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="label_new_pass">
          <property name="text">
           <string>new pass</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLineEdit" name="lineEdit_new_pass_1"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_new_pass_2"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_change_pass_word">
      <property name="text">
       <string>change pass</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
